### ISSUE TITLE
checkpoint shown as from 2014 (only display bug or more?)

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [gridcoin-community/Gridcoin-Research#417]
Still same in Gridcoin 3.5.9.1 (since May at least, see (A)), so reporting as a new issue:

    getcheckpoint

    {
    "synccheckpoint" : "000005a247b397eadfefa58e872bc967c2614797bdc8d4d0e6b09fea5c191599",
    "height" : 0,
    "timestamp" : "2014-10-11 13:22:57 UTC",
    "policy" : "strict"
    }

I know from bug #144 that in Nov 2016 it showed block height 278000 (from 2015 July), which was too old, but at least better than zero.
Please also comment on implications about
* security (e.g. #342, ...), 
* syncing speed for new clients, 
* ...

(A) 

https://github.com/gridcoin/Gridcoin-Research/issues/144#issuecomment-303007434

the synccheckpoint in 3.5.8.9 is reset :-(
wallet says: block 0 from 2014 October 11

This is probably the reason why some people got onto a fork when they synced from 0. I wonder in which version it was reset ?

https://github.com/gridcoin/Gridcoin-Research/issues/144#issuecomment-303617619
@Foggyx420
its not working well at all. seems like more and more things out of whack i agree but CheckHardened which is called on when accepting a block is working just fine. Its called to check if a checkpoint hash matches the blockhash and if so its accepted as checkpoint. if the hash doesn't match of course it stops it right there. and if the block being checked is not in the map it passes. i wouldn't panic but there is defiantly issues and it seems they may of been here for a very long time.

**Comment by iFoggz** [comment]
I looked at it more and it seems to work on syncing up from 0 without a problem. but if ur beyond the checkpoints it says that. but also I think I experienced once I stopped during a sync and reloaded and it said that thou the printf messages I had added showed it passing through it as it should. so idk. some issue but I don't believe its causing a problem per say

**Comment by denravonska** [comment]
Fixed in 3.7.0.0:

```
$ ./gridcoinresearchd getcheckpoint
{
    "synccheckpoint" : "4be0afdb9273d232de0bc75b572d8bcfaa146d9efdbe4a4f1ab775334f175b0f",
    "height" : 950000,
    "timestamp" : "2017-07-03 14:29:36 UTC"
}
```
